dot/parachain/types/bitvec_test.go

```diff
@@ -634,3 +634,18 @@ func TestBitVec_Mask(t *testing.T) {
 		})
 	}
 }
+
+func TestBitVec_Clone(t *testing.T) {
+	t.Parallel()
+
+	a, err := NewBitVec([]bool{true, true, false, true, true})
+	require.NoError(t, err)
+
+	b := a.Clone()
+	require.Equal(t, a.Bits(), b.Bits())
```

Comment:
As it's a deep copy, Let's ensure the another field is also same.
```suggestion
	require.Equal(t, a.Bits(), b.Bits())
	require.Equal(t, a.len, b.len)
```